conf/stats.conf.defaults

```diff
@@ -202,3 +202,51 @@ api_compile=$.items[*].network, $.items[*].percentused
 interval=60s
 randomize=true
 management=false
+
+[metric 'Error in packetfence.log']
+type=tail_file
+file=/usr/local/pf/logs/packetfence.log
+match=ERROR,httpd.portal.*Unable to match MAC address to IP.*ip2mac.*
+statsd_ns=source.packetfence.packetfence_log.error,source.packetfence.packetfence_log.portal.ip2mac
+
+[metric 'DHCP messages in pfdhcp.log']
+type=tail_file
+file=/usr/local/pf/logs/pfdhcp.log
+match=DHCPDISCOVER,DHCPOFFER,DHCPREQUEST,DHCPACK,DHCPNAK,DHCPDECLINE,DHCPRELEASE,DHCPINFORM
+statsd_ns=source.packetfence.pfdhcp_log.dhcpdiscover,source.packetfence.pfdhcp_log.dhcpoffer,source.packetfence.pfdhcp_log.dhcprequest,source.packetfence.pfdhcp_log.dhcpack,source.packetfence.pfdhcp_log.dhcpnak,source.packetfence.pfdhcp_log.dhcpdecline,source.packetfence.pfdhcp_log.dhcprelease,source.packetfence.pfdhcp_log.dhcpinform
+
+[metric 'RADIUS load_balancer.log']
```

Comment:
To be more precise, can we call it *Errors in radius-load_balancer.log*?

Reply:
It's can be something else than error, can we use events instead ?

Reply:
Of course!